bin/slackin

```diff
@@ -6,7 +6,7 @@ var slackin = require('./../node').default;
 
 args
   .option(['p', 'port'], 'Port to listen on [$PORT or 3000]', require('hostenv').PORT || process.env.PORT || 3000)
-  .option(['h', 'hostname'], 'Hostname to listen on [$HOSTNAME or 0.0.0.0]', require('hostenv').HOSTNAME || process.env.WEBSITE_HOSTNAME || '0.0.0.0')
+  .option(['H', 'hostname'], 'Hostname to listen on [$HOSTNAME or 0.0.0.0]', require('hostenv').HOSTNAME || process.env.WEBSITE_HOSTNAME || '0.0.0.0')
```

Comment:
I think the label of the flags just comes down to personal taste in the end. My suggestion would be to do something similar to what MySQL and Redis do, where `-h` is the hostname flag and `-?` is the help flag.

That would have the benefit of keeping `-h` backwards compatible, and since the help page is shown when incomplete arguments are passed to the slackin command, anyone who would type in `slackin -h` *should* still get shown the help page.